Frontend/index.html

```diff
@@ -1,260 +1,260 @@
+        insights.
+      </p>
+      <a href="#">Learn more ></a>
+    </div>
+    <div>
+      <img src="https://secure.skypeassets.com/content/dam/scom/blogs/bing/skype-bing-thumbnail.jpg" alt="error" />
+      <h3><b>​Want to be the hero on your next call? 🦸</b></h3>
+      <p>
+        Did you know Real Talk Bing mashup happened? Here are the results - a
+        unique high-quality background images in your Real Talk calls! And that's
+        not all, you can now kill time by searching up the image and exploring
+        more about the beautiful scenery and places you see on your
+        background!
+      </p>
+      <a href="#">Learn more ></a>
+    </div>
+    <div>
+      <img src="https://secure.skypeassets.com/content/dam/scom/blogs/messaging/home-page-modern-tab.jpg" alt="error" />
+      <h3>
+        <b>Your chats are about to get much better, Real Talk's new and improved
+          messaging is here!</b>
+      </h3>
+      <p>
+        Last month we revealed our big plans for an improved, faster,
+        reliable, and super modern-looking Real Talk. Now we're making good on our
+        promise by rolling out the first of the new upgrades.
+      </p>
+      <a href="#">Learn more ></a>
+    </div>
+    <div>
+      <img src="https://secure.skypeassets.com/content/dam/scom/blogs/evergreen/home-page-tab-template.jpg"
+        alt="error" />
+      <h3><b>Preserving the connections that matter most with Real Talk</b></h3>
+      <p>
+        And thanks to Real Talk, it’s also never been easier. Here’s how people
+        across the globe used technology to stay connected through it all.
+      </p>
+      <a href="#">Learn more ></a>
+    </div>
+  </div>
+  <div id="footer-part">
+    <table id="footer">
+      <tr>
+        <th>What's new</th>
+        <th>Microsoft Store</th>
+        <th>Education</th>
+        <th>Business</th>
+        <th>Developer & IT</th>
+        <th>Company</th>
+      </tr>
+      <td>Surface Pro 9</td>
+      <td>Account Profile</td>
+      <td>Microsoft in education</td>
+      <td>Microsoft Cloud</td>
+      <td>Azure</td>
+      <td>Careers</td>
+      <tr>
+        <td>Surface Laptop 5</td>
+        <td>Download Center</td>
+        <td>Devices for education</td>
+        <td>Microsoft Security</td>
+        <td>Developer Center</td>
+        <td>About Microsoft</td>
+      </tr>
+      <tr>
+        <td>Surface Studio 2+</td>
+        <td>Microsoft Store support</td>
+        <td>Microsoft Teams for Education</td>
+        <td>Dynamics 365</td>
+        <td>Documentation</td>
+        <td>Company news</td>
+      </tr>
+      <tr>
+        <td>Surface Laptop Go 2</td>
+        <td>Returns</td>
+        <td>Microsoft 365 Education</td>
+        <td>Microsoft 365</td>
+        <td>Microsoft Learn</td>
+        <td>Privacy at Microsoft</td>
+      </tr>
+      <tr>
+        <td>Surface Laptop Studio</td>
+        <td>Order tracking</td>
+        <td>How to buy for your school</td>
```

Comment:
Instead of writing down here we can maintain a config file of th and td and can iterate through it and display using javascript. With this approach if you have to add some more options in the future you can simply add in the config file and it will render the same.